services/movie.py

```diff
@@ -0,0 +1,34 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    movies = Movie.objects.all()
+    if not genres_ids and not actors_ids:
+        return movies
+    elif genres_ids and actors_ids:
+        return (movies.filter
+                (genres__id__in=genres_ids).filter(actors__id__in=actors_ids))
+    elif genres_ids:
+        return movies.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        return movies.filter(actors__id__in=actors_ids)
```

Comment:
Add one return statement here and leave only two if statements to filter additionally by `genres_ids` or `actors_ids` in case they are present. Remember that querysets are lazy